android-sdk/src/main/java/com/mobilecoin/lib/MobileCoinClient.java

```diff
@@ -192,6 +192,11 @@ public MobileCoinClient(
         this.blockchainClient = blockchainClient;
     }
 
+    @NonNull
```

Comment:
the package-private constructor allows the `untrustedClient` to be null on line 177